flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorITCase.java

```diff
@@ -442,6 +448,41 @@ public void testDynamicPartWithOrderBy() throws Exception {
         }
     }
 
+    @Test
+    public void testLocationWithComma() throws Exception {
+        TableEnvironment tableEnv = getTableEnvWithHiveCatalog();
+        File location = tempFolder.newFolder(",tbl1,location,");
+        try {
+            // test table location
+            tableEnv.executeSql(
+                    String.format(
+                            "create table tbl1 (x int) location '%s'", location.getAbsolutePath()));
+            tableEnv.executeSql("insert into tbl1 values (1),(2)").await();
+            List<Row> results =
+                    CollectionUtil.iteratorToList(
+                            tableEnv.executeSql("select * from tbl1").collect());
+            assertEquals("[+I[1], +I[2]]", results.toString());
+            // test partition location
+            tableEnv.executeSql("create table tbl2 (x int) partitioned by (p string)");
+            location = tempFolder.newFolder(",");
+            tableEnv.executeSql(
+                    String.format(
+                            "alter table tbl2 add partition (p='a') location '%s'",
```

Comment:
The partition path should have comma?

Reply:
The location path has comma and hive's partition value doesn't have to be the same as the location path. But I can also add a test case where the partition value also contains comma.

Reply:
According to the jira description, I was assuming that you want to address both of these

Reply:
It's the location path that causes the issue. But I added a test case where both partition value and path contain comma anyways.